reports/package.json

```diff
@@ -10,10 +10,17 @@
   },
   "engines": {
     "npm": ">=7.0.0",
-    "node": ">=16.0.0"
+    "node": ">=16.14.0"
   },
   "type": "module",
   "dependencies": {
-    "@evidence-dev/evidence": "^14.0.0"
+    "@evidence-dev/evidence": "15.0.1",
+    "@evidence-dev/preprocess": "2.2.0",
+    "@evidence-dev/components": "2.2.1"
+  },
+  "overrides": {
+    "jsonwebtoken": "9.0.0",
+    "trim@<0.0.3": ">0.0.3",
+    "sqlite3": "5.1.5"
```

Comment:
@hughess - Can you provide context for these overrides? Should these be considered temporary or permanent?

Reply:
@Winterhart would be the best person to give that context!

Reply:
- The `sqlite3` is temporary. It will soon be not necessary. 
- The `jsonwebtoken` is temporary. It will soon be not necessary. It comes from `snowflake-sdk`
- The `trim` will stay longer as it depends on `remark-parse` and we don't have an upgrade planned right now.

The goal is to keep the `template` at 0 CVE.